repo/packages/A/artifactory/3/config.json

```diff
@@ -0,0 +1,203 @@
+{
+  "type": "object",
+  "properties": {
+    "service": {
+      "description": "Artifactory service properties.",
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "The name of this DC/OS service.",
+          "type": "string",
+          "default": "artifactory"
+        },
+        "cpus": {
+          "description": "CPU shares to allocate to this instance.",
+          "type": "number",
+          "default": 2.0,
+          "minimum": 1.0
+        },
+        "mem": {
+          "description": "Memory to allocate to this instance.",
+          "type": "number",
+          "default": 2048.0,
+          "minimum": 1024.0
+        },
+        "nodes": {
+          "description": "Number of Artifactory instances. By default, this is 1.",
+          "type": "integer",
+          "default": 1
+        },
+        "licenses": {
```

Comment:
There is no OSS version?

Reply:
Thank for you comments. We would like to continue with our license based installation.